crates/forge/src/lib.rs

```diff
@@ -31,7 +30,7 @@ pub mod scarb;
 pub mod test_case_summary;
 
 mod fuzzer;
-mod running;
+pub mod running;
```

Comment:
Does it need to be pub? We can explicitly reexport some functions with `pub use ...` if they are needed somewhere.

Reply:
Will try that, but I believe they have to be pub in the first place

Reply:
Yep, it doesn't work, have to be pub in the first place
https://stackoverflow.com/questions/69275034/what-is-the-difference-between-use-and-pub-use